docs/README.md

```diff
@@ -0,0 +1,44 @@
+# Why are we building OLM v1?
+
+Operator-lifecycle-manager's mission has been to manage the lifecycle of cluster extensions centrally and declaratively on Kubernetes clusters. Its purpose has always been to make installing, 
+running, and updating functional extensions to the cluster easy, safe, and reproducible for cluster administrators and PaaS administrators, throughout the lifecycle of the underlying cluster. 
+
+OLM so far has focused on providing unique support for these specific needs for a particular type of cluster extension, which have been coined as [operators](https://operatorhub.io/what-is-an-operator#:~:text=is%20an%20Operator-,What%20is%20an%20Operator%20after%20all%3F,or%20automation%20software%20like%20Ansible.). 
+Operators were classified as one or more Kubernetes controllers, shipping with one or more API extensions (CustomResourceDefinitions) to provide additional functionality to the cluster. 
+Over the last few years of running OLM in production cluster, it became apparent that there's an appetite to deviate from this coupling of CRDs and controllers, to encompass the lifecycling 
+of extentions that are not just operators.
+
+OLM has been assisting in defining a lifecycle for these extensions in which they get installed, potentially causing other extensions to be installed as well as dependencies, with a limited set of 
```

Comment:
Nit/nice-to-have: IMO it would be nice to include a visual depiction of what the lifecycle is

Reply:
Perhaps:
```
OLM has been of assistance in defining...
```
Having two "ing" verbs in a row seems odd.

Reply:
I would suggest making each phrase in this sentence into a list item, showing the iteration of the lifecycle.

Reply:
Changed the sentence to be a list of items. Keeping the conversation as unresolved so that we can create a follow up issue for creating a visual depiction of the lifecycle